Wikipedia/Code/Session.swift

```diff
@@ -667,8 +677,7 @@ class SessionDelegate: NSObject, URLSessionDelegate, URLSessionDataDelegate {
         if let error = error as NSError? {
             if error.domain != NSURLErrorDomain || error.code != NSURLErrorCancelled {
                 
-                if let request = task.currentRequest,
-                request.cachePolicy != .reloadIgnoringLocalCacheData,
```

Comment:
Also fixes the regression issue mentioned in https://github.com/wikimedia/wikipedia-ios/pull/3479.